components/properties/src/props.rs

```diff
@@ -39,6 +41,41 @@ pub enum EnumeratedProperty {
     InvalidCode = -1, // TODO(#1160) - taken from ICU4C UProperty::UCHAR_INVALID_CODE
 }
 
+/// Enumerated property Bidi_Class
+///
+///
+///
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[repr(transparent)]
+#[zerovec::make_ule(BidiClassULE)]
+pub struct BidiClass(pub u16);
```

Comment:
Nit: If there are only 22 types of BidiClass, then make this a `u8` instead of `u16` and cut data size in half.

Reply:
It's also important to match the ICU value width. If we use a type that is shorter than ICU's trie value width, then we will truncate (corrupt) the data into something meaningless, and if we use a type that is wider than ICU's value width, we are wasting space.

Reply:
Oh, hmm. I didn't realize `valueWidth` was 2 from the ICU4C Trie. I wonder why that is the case. We should investigate.

Reply:
OK cool, so it actually is correct to use `u8` here. Thanks for the clarification @echeran !

Reply:
Thanks @echeran  for the illustration